src/main/cpp/auton/ZoneParams.cpp

```diff
@@ -23,7 +23,10 @@
 ZoneParams::ZoneParams(int xgrid1,
```

Comment:
shouldn't we be using the zone enums and the chassisoptionenums instead of ints?